pandas/algos.pyx

```diff
@@ -1348,6 +1348,56 @@ cdef inline float64_t _median_linear(float64_t* a, int n):
     return result
 
 
+cdef int64_t _checked_add_scalars(int64_t a, int64_t b) except -1:
```

Comment:
Argh.  Good point.  However, @jorisvandenbossche 's vectorization should make this moot.